test/integration/smoke/test_ospf_zone_config.py

```diff
@@ -0,0 +1,165 @@
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
+        
+        
+        cmd = vpcQuaggaConfig.vpcQuaggaConfigCmd()
+        cmd.zoneid = self.zone.id
+        response = self.apiclient.vpcQuaggaConfig(cmd)
+        self.debug("Response quaggaEnabled: %s" % response.quaggaEnabled)
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
+        
+        cmd.zoneid = self.zone.id
+        
+        
+        self.assertEqual(
+                         response.quaggaEnabled, "true"
+                         )
+        
+        
+        self.assertEqual(
+                         response.quaggaPassword, "6P33lsadEbmiJ7ZN7gK8vylOY2dTza3rAuY8UFboH7I=:Xb1G4VhzBkSXmTa8vDqExQQo8PiIUUSNtpxmdOkX1z8="
+                         )
+        
+        self.assertEqual(
+                         response.ospfSuperCIDR, "200.100.0.0/18,216.100.0.0/20"
+                         )
+        
+        return
+
+        
+    # Check vpcQuaggaConfigCmd positive test case
+    #VPCQuaggaConfigUpdateCmd [zoneid=1, quaggaProtocol=null, ospfArea=null, quaggaHelloInterval=null, quaggaDeadInterval=null, quaggaRetransmitInterval=null, quaggaTransitDelay=null, quaggaAuthentication=null, quaggaPassword=junk, ospfSuperCIDR=null, quaggaEnabled=null]
+    @attr(tags=["smoke", "advanced"], required_hardware="false")
+    def test_02_quaggaConfig(self):
+        
+        cmd = vpcQuaggaConfig.vpcQuaggaConfigCmd()
+        cmd.zoneid = self.zone.id
+        response = self.apiclient.vpcQuaggaConfig(cmd)
+        self.debug("Response quaggaEnabled: %s" % response.quaggaEnabled)
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
+        
+        
+        cmd = vpcQuaggaConfigUpdate.vpcQuaggaConfigUpdateCmd()
+        cmd.zoneid = self.zone.id
+        cmd.ospfSuperCIDR = "200.100.0.0/18,216.100.0.0/20"
+        cmd.quaggaEnabled = "false"
+        cmd.quaggaPassword = "6P33lsadEbmiJ7ZN7gK8vylOY2dTza3rAuY8UFboH7I=:Xb1G4VhzBkSXmTa8vDqExQQo8PiIUUSNtpxmdOkX1z8="
+        response = self.apiclient.vpcQuaggaConfigUpdate(cmd)
+        self.debug("Response quaggaEnabled: %s" % response.quaggaEnabled)
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
+        
+        
+        cmd = vpcQuaggaConfig.vpcQuaggaConfigCmd()
+        cmd.zoneid = self.zone.id
+        response = self.apiclient.vpcQuaggaConfig(cmd)
+        self.debug("Response quaggaEnabled: %s" % response.quaggaEnabled)
+        self.debug("Response quaggaPassword: %s" % response.quaggaPassword)
+        self.debug("Response ospfSuperCIDR: %s" % response.ospfSuperCIDR)
```

Comment:
Lines 119-145 and lines 71-95 appear similar.  Consider extracting to a private method.